source/default/_patterns/00-protons/tailwind.tokens.css

```diff
@@ -14,10 +14,81 @@ Example: Google Remote Fonts
 Serif - Google Fonts - Oswald
 Sans-Serif - Google Fonts - Open Sans
 Monospace - Google Fonts - Source Code Pro
-*/
+*/ 
 
 @import url(https://fonts.googleapis.com/css?family=OpenSans|Oswald|SourceCodePro&display=swap);
 
+
+/* Button Variations */
+@layer components {
+  .btn {
+    @apply text-center align-middle select-none mb-4 font-semibold py-1.5 px-3 text-base;
+  }
```

Comment:
makes sense although i wonder if we should prefix or use a different word since that's the bootstrap standard. It could be confusing for someone looking at it.

Reply:
I guess I was trying to match what we've already done in _button.css as far as naming and styles...so that it doesn't have a site-wide impact (e.g. .btn in use in text editor). What do you think, am I thinking about it correctly?

Reply:
that's an interesting take. I had not considered its backward compatibility. It makes sense, I will read and think a bit more to see if there's more info about it.

Reply:
Thanks for checking, @bmartinez287, just lmk if any changes are needed on this. We could always leave the bootstrap button classes in the css file, just wasn't sure about maintaining in two places. We can also take out these and just keep the button styles in .css if that's more logical with how we have things set up.